### PR TITLE
Remove invoice feature

### PR DESCRIPTION
**Pull request by ashkan18**
# Change
We no longer support conversation invoice. This PR removes invoice preview portion of conversation, messages.

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>
<b>🃏 FAIL</b> in <a href="https://github.com/artsy/emission/blob/remove-invoice/src/lib/Components/Inbox/Conversations/__tests__/Message-tests.tsx">Message-tests.tsx</a>


<li>
<a href='https://github.com/artsy/emission/blob/remove-invoice/src/lib/Components/Inbox/Conversations/__tests__/Message-tests.tsx#L52'>looks correct when rendered</a>
<br/>
Snapshot name: `looks correct when rendered 1`- Snapshot+ Received@@ -117,436 +117,10 @@ > A YEAR AGO </Text> </View> <View- style={- Array [- Object {- "marginBottom": 10,- "width": 295,- },- ]- }- >- <View- style={- Array [- Object {- "borderColor": "#e5e5e5",- "borderWidth": 1,- "flexDirection": "row",- "height": 72,- },- ]- }- >- <Image- source={- Object {- "testUri": "../../../images/payment_request.png",- }- }- style={- Array [- Object {- "marginBottom": 12,- "marginLeft": 12,- "marginRight": 12,- "marginTop": 12,- "resizeMode": "contain",- "width": 40,- },- ]- }- />- <View- style={- Array [- Object {- "alignSelf": "center",- "flexBasis": 0,- "flexDirection": "column",- "flexGrow": 1,- "flexShrink": 1,- },- ]- }- >- <Text- style={- Array [- Object {- "fontFamily": "ReactNativeAGaramondPro-Regular",- "fontSize": 14,- },- ]- }- >- Payment request- </Text>- <Text- style={- Array [- Object {- "fontFamily": "ReactNativeAGaramondPro-Regular",- "fontSize": 14,- "fontWeight": "bold",- },- ]- }- >- $420- </Text>- </View>- <View- style={- Array [- Object {- "alignSelf": "center",- "height": 25,- "marginRight": 15,- "width": 78,- },- ]- }- >- <View- accessible={true}- flexDirection="row"- focusable={true}- onClick={[Function]}- onResponderGrant={[Function]}- onResponderMove={[Function]}- onResponderRelease={[Function]}- onResponderTerminate={[Function]}- onResponderTerminationRequest={[Function]}- onStartShouldSetResponder={[Function]}- style={- Array [- Object {- "display": "flex",- "flexDirection": "row",- },- ]- }- >- <View- block={true}- onPress={[Function]}- px={1}- size="small"- style={- Array [- Object {- "alignItems": "center",- "borderRadius": 3,- "borderWidth": 1,- "justifyContent": "center",- "paddingLeft": 10,- "paddingRight": 10,- "position": "relative",- "width": "100%",- },- Object {- "backgroundColor": "rgba(0, 0, 0, 1)",- "borderColor": "rgba(0, 0, 0, 1)",- "color": "rgba(255, 255, 255, 1)",- "height": 26,- "opacity": 1,- },- ]- }- theme={- Object {- "borders": Array [- "1px solid",- "2px solid",- ],- "breakpoints": Array [- 768,- 900,- 1024,- 1192,- ],- "colors": Object {- "black10": "#E5E5E5",- "black100": "#000",- "black30": "#C2C2C2",- "black5": "#F8F8F8",- "black60": "#666",- "black80": "#333",- "green100": "#0EDA83",- "purple100": "#6E1EFF",- "purple30": "#D3BBFF",- "purple5": "#F8F3FF",- "red100": "#F7625A",- "white100": "#FFF",- "yellow10": "#FDF7E8",- "yellow100": "#F1AF1B",- "yellow30": "#FAE7BA",- },- "fontFamily": Object {- "display": Object {- "regular": "AvantGardeGothicITC",- },- "sans": Object {- "italic": "Unica77LL-Italic",- "medium": "Unica77LL-Medium",- "mediumItalic": "Unica77LL-MediumItalic",- "regular": "Unica77LL-Regular",- },- "serif": Object {- "italic": "ReactNativeAGaramondPro-Italic",- "regular": "ReactNativeAGaramondPro-Regular",- "semibold": "ReactNativeAGaramondPro-Semibold",- },- },- "grid": Object {- "breakpoints": Object {- "lg": 1024,- "md": 900,- "sm": 768,- "xl": 1192,- "xs": 767,- },- "col": Object {- "padding": 0,- },- "container": Object {- "padding": 0,- },- "row": Object {- "padding": 0,- },- },- "mediaQueries": Object {- "hover": "not all and (pointer: coarse), not all and (-moz-touch-enabled: 1)",- "lg": "(min-width: 1024px) and (max-width: 1191px)",- "md": "(min-width: 900px) and (max-width: 1023px)",- "sm": "(min-width: 768px) and (max-width: 899px)",- "xl": "(min-width: 1192px)",- "xs": "(max-width: 767px)",- },- "space": Object {- "0.3": 3,- "0.5": 5,- "1": 10,- "12": 120,- "18": 180,- "2": 20,- "3": 30,- "4": 40,- "6": 60,- "9": 90,- },- "typeSizes": Object {- "display": Object {- "2": Object {- "fontSize": 10,- "lineHeight": 12,- },- "3t": Object {- "fontSize": 12,- "lineHeight": 16,- },- "4t": Object {- "fontSize": 14,- "lineHeight": 18,- },- "5t": Object {- "fontSize": 16,- "lineHeight": 20,- },- "6": Object {- "fontSize": 18,- "lineHeight": 22,- },- "8": Object {- "fontSize": 22,- "lineHeight": 24,- },- },- "sans": Object {- "0": Object {- "fontSize": 8,- "lineHeight": 8,- },- "1": Object {- "fontSize": 10,- "lineHeight": 14,- },- "10": Object {- "fontSize": 42,- "lineHeight": 50,- },- "12": Object {- "fontSize": 60,- "lineHeight": 66,- },- "14": Object {- "fontSize": 80,- "lineHeight": 84,- },- "16": Object {- "fontSize": 100,- "lineHeight": 104,- },- "2": Object {- "fontSize": 12,- "lineHeight": 16,- },- "3": Object {- "fontSize": 14,- "lineHeight": 24,- },- "3t": Object {- "fontSize": 14,- "lineHeight": 20,- },- "4": Object {- "fontSize": 16,- "lineHeight": 26,- },- "4t": Object {- "fontSize": 16,- "lineHeight": 22,- },- "5": Object {- "fontSize": 18,- "lineHeight": 30,- },- "5t": Object {- "fontSize": 18,- "lineHeight": 26,- },- "6": Object {- "fontSize": 22,- "lineHeight": 30,- },- "8": Object {- "fontSize": 28,- "lineHeight": 36,- },- },- "serif": Object {- "1": Object {- "fontSize": 12,- "lineHeight": 16,- },- "10": Object {- "fontSize": 44,- "lineHeight": 50,- },- "12": Object {- "fontSize": 60,- "lineHeight": 70,- },- "2": Object {- "fontSize": 14,- "lineHeight": 18,- },- "3": Object {- "fontSize": 16,- "lineHeight": 24,- },- "3t": Object {- "fontSize": 16,- "lineHeight": 20,- },- "4": Object {- "fontSize": 18,- "lineHeight": 26,- },- "4t": Object {- "fontSize": 18,- "lineHeight": 22,- },- "5": Object {- "fontSize": 22,- "lineHeight": 32,- },- "5t": Object {- "fontSize": 22,- "lineHeight": 28,- },- "6": Object {- "fontSize": 26,- "lineHeight": 32,- },- "8": Object {- "fontSize": 32,- "lineHeight": 38,- },- },- },- }- }- variant="primaryBlack"- width="100%"- >- <View- style={- Array [- Object {- "alignItems": "center",- "display": "flex",- "flexDirection": "row",- "height": "100%",- "justifyContent": "center",- "position": "absolute",- "width": "100%",- },- ]- }- >- <Text- color="#FFF"- fontFamily="Unica77LL-Medium"- fontSize="12px"- lineHeight="16px"- style={- Array [- Object {- "color": "#FFF",- "fontFamily": "Unica77LL-Medium",- "fontSize": 12,- "lineHeight": 16,- },- Object {},- ]- }- >- Pay- </Text>- </View>- <Text- fontFamily="Unica77LL-Medium"- fontSize="12px"- lineHeight="16px"- role="presentation"- style={- Array [- Object {- "fontFamily": "Unica77LL-Medium",- "fontSize": 12,- "lineHeight": 16,- },- Object {- "opacity": 0,- },- ]- }- >- Pay- </Text>- </View>- </View>- </View>- </View>- </View>- <View linkStyle={ Object { "color": "#6e1fff", "textDecorationLine": "underline", }

<details>
<summary>Full message</summary>
<pre><code>
Error: expect(received).toMatchSnapshot()

Snapshot name: `looks correct when rendered 1`

- Snapshot
+ Received

@@ -117,436 +117,10 @@
          >
            A YEAR AGO
          </Text>
        </View>
        <View
-         style={
-           Array [
-             Object {
-               "marginBottom": 10,
-               "width": 295,
-             },
-           ]
-         }
-       >
-         <View
-           style={
-             Array [
-               Object {
-                 "borderColor": "#e5e5e5",
-                 "borderWidth": 1,
-                 "flexDirection": "row",
-                 "height": 72,
-               },
-             ]
-           }
-         >
-           <Image
-             source={
-               Object {
-                 "testUri": "../../../images/payment_request.png",
-               }
-             }
-             style={
-               Array [
-                 Object {
-                   "marginBottom": 12,
-                   "marginLeft": 12,
-                   "marginRight": 12,
-                   "marginTop": 12,
-                   "resizeMode": "contain",
-                   "width": 40,
-                 },
-               ]
-             }
-           />
-           <View
-             style={
-               Array [
-                 Object {
-                   "alignSelf": "center",
-                   "flexBasis": 0,
-                   "flexDirection": "column",
-                   "flexGrow": 1,
-                   "flexShrink": 1,
-                 },
-               ]
-             }
-           >
-             <Text
-               style={
-                 Array [
-                   Object {
-                     "fontFamily": "ReactNativeAGaramondPro-Regular",
-                     "fontSize": 14,
-                   },
-                 ]
-               }
-             >
-               Payment request
-             </Text>
-             <Text
-               style={
-                 Array [
-                   Object {
-                     "fontFamily": "ReactNativeAGaramondPro-Regular",
-                     "fontSize": 14,
-                     "fontWeight": "bold",
-                   },
-                 ]
-               }
-             >
-               $420
-             </Text>
-           </View>
-           <View
-             style={
-               Array [
-                 Object {
-                   "alignSelf": "center",
-                   "height": 25,
-                   "marginRight": 15,
-                   "width": 78,
-                 },
-               ]
-             }
-           >
-             <View
-               accessible={true}
-               flexDirection="row"
-               focusable={true}
-               onClick={[Function]}
-               onResponderGrant={[Function]}
-               onResponderMove={[Function]}
-               onResponderRelease={[Function]}
-               onResponderTerminate={[Function]}
-               onResponderTerminationRequest={[Function]}
-               onStartShouldSetResponder={[Function]}
-               style={
-                 Array [
-                   Object {
-                     "display": "flex",
-                     "flexDirection": "row",
-                   },
-                 ]
-               }
-             >
-               <View
-                 block={true}
-                 onPress={[Function]}
-                 px={1}
-                 size="small"
-                 style={
-                   Array [
-                     Object {
-                       "alignItems": "center",
-                       "borderRadius": 3,
-                       "borderWidth": 1,
-                       "justifyContent": "center",
-                       "paddingLeft": 10,
-                       "paddingRight": 10,
-                       "position": "relative",
-                       "width": "100%",
-                     },
-                     Object {
-                       "backgroundColor": "rgba(0, 0, 0, 1)",
-                       "borderColor": "rgba(0, 0, 0, 1)",
-                       "color": "rgba(255, 255, 255, 1)",
-                       "height": 26,
-                       "opacity": 1,
-                     },
-                   ]
-                 }
-                 theme={
-                   Object {
-                     "borders": Array [
-                       "1px solid",
-                       "2px solid",
-                     ],
-                     "breakpoints": Array [
-                       768,
-                       900,
-                       1024,
-                       1192,
-                     ],
-                     "colors": Object {
-                       "black10": "#E5E5E5",
-                       "black100": "#000",
-                       "black30": "#C2C2C2",
-                       "black5": "#F8F8F8",
-                       "black60": "#666",
-                       "black80": "#333",
-                       "green100": "#0EDA83",
-                       "purple100": "#6E1EFF",
-                       "purple30": "#D3BBFF",
-                       "purple5": "#F8F3FF",
-                       "red100": "#F7625A",
-                       "white100": "#FFF",
-                       "yellow10": "#FDF7E8",
-                       "yellow100": "#F1AF1B",
-                       "yellow30": "#FAE7BA",
-                     },
-                     "fontFamily": Object {
-                       "display": Object {
-                         "regular": "AvantGardeGothicITC",
-                       },
-                       "sans": Object {
-                         "italic": "Unica77LL-Italic",
-                         "medium": "Unica77LL-Medium",
-                         "mediumItalic": "Unica77LL-MediumItalic",
-                         "regular": "Unica77LL-Regular",
-                       },
-                       "serif": Object {
-                         "italic": "ReactNativeAGaramondPro-Italic",
-                         "regular": "ReactNativeAGaramondPro-Regular",
-                         "semibold": "ReactNativeAGaramondPro-Semibold",
-                       },
-                     },
-                     "grid": Object {
-                       "breakpoints": Object {
-                         "lg": 1024,
-                         "md": 900,
-                         "sm": 768,
-                         "xl": 1192,
-                         "xs": 767,
-                       },
-                       "col": Object {
-                         "padding": 0,
-                       },
-                       "container": Object {
-                         "padding": 0,
-                       },
-                       "row": Object {
-                         "padding": 0,
-                       },
-                     },
-                     "mediaQueries": Object {
-                       "hover": "not all and (pointer: coarse), not all and (-moz-touch-enabled: 1)",
-                       "lg": "(min-width: 1024px) and (max-width: 1191px)",
-                       "md": "(min-width: 900px) and (max-width: 1023px)",
-                       "sm": "(min-width: 768px) and (max-width: 899px)",
-                       "xl": "(min-width: 1192px)",
-                       "xs": "(max-width: 767px)",
-                     },
-                     "space": Object {
-                       "0.3": 3,
-                       "0.5": 5,
-                       "1": 10,
-                       "12": 120,
-                       "18": 180,
-                       "2": 20,
-                       "3": 30,
-                       "4": 40,
-                       "6": 60,
-                       "9": 90,
-                     },
-                     "typeSizes": Object {
-                       "display": Object {
-                         "2": Object {
-                           "fontSize": 10,
-                           "lineHeight": 12,
-                         },
-                         "3t": Object {
-                           "fontSize": 12,
-                           "lineHeight": 16,
-                         },
-                         "4t": Object {
-                           "fontSize": 14,
-                           "lineHeight": 18,
-                         },
-                         "5t": Object {
-                           "fontSize": 16,
-                           "lineHeight": 20,
-                         },
-                         "6": Object {
-                           "fontSize": 18,
-                           "lineHeight": 22,
-                         },
-                         "8": Object {
-                           "fontSize": 22,
-                           "lineHeight": 24,
-                         },
-                       },
-                       "sans": Object {
-                         "0": Object {
-                           "fontSize": 8,
-                           "lineHeight": 8,
-                         },
-                         "1": Object {
-                           "fontSize": 10,
-                           "lineHeight": 14,
-                         },
-                         "10": Object {
-                           "fontSize": 42,
-                           "lineHeight": 50,
-                         },
-                         "12": Object {
-                           "fontSize": 60,
-                           "lineHeight": 66,
-                         },
-                         "14": Object {
-                           "fontSize": 80,
-                           "lineHeight": 84,
-                         },
-                         "16": Object {
-                           "fontSize": 100,
-                           "lineHeight": 104,
-                         },
-                         "2": Object {
-                           "fontSize": 12,
-                           "lineHeight": 16,
-                         },
-                         "3": Object {
-                           "fontSize": 14,
-                           "lineHeight": 24,
-                         },
-                         "3t": Object {
-                           "fontSize": 14,
-                           "lineHeight": 20,
-                         },
-                         "4": Object {
-                           "fontSize": 16,
-                           "lineHeight": 26,
-                         },
-                         "4t": Object {
-                           "fontSize": 16,
-                           "lineHeight": 22,
-                         },
-                         "5": Object {
-                           "fontSize": 18,
-                           "lineHeight": 30,
-                         },
-                         "5t": Object {
-                           "fontSize": 18,
-                           "lineHeight": 26,
-                         },
-                         "6": Object {
-                           "fontSize": 22,
-                           "lineHeight": 30,
-                         },
-                         "8": Object {
-                           "fontSize": 28,
-                           "lineHeight": 36,
-                         },
-                       },
-                       "serif": Object {
-                         "1": Object {
-                           "fontSize": 12,
-                           "lineHeight": 16,
-                         },
-                         "10": Object {
-                           "fontSize": 44,
-                           "lineHeight": 50,
-                         },
-                         "12": Object {
-                           "fontSize": 60,
-                           "lineHeight": 70,
-                         },
-                         "2": Object {
-                           "fontSize": 14,
-                           "lineHeight": 18,
-                         },
-                         "3": Object {
-                           "fontSize": 16,
-                           "lineHeight": 24,
-                         },
-                         "3t": Object {
-                           "fontSize": 16,
-                           "lineHeight": 20,
-                         },
-                         "4": Object {
-                           "fontSize": 18,
-                           "lineHeight": 26,
-                         },
-                         "4t": Object {
-                           "fontSize": 18,
-                           "lineHeight": 22,
-                         },
-                         "5": Object {
-                           "fontSize": 22,
-                           "lineHeight": 32,
-                         },
-                         "5t": Object {
-                           "fontSize": 22,
-                           "lineHeight": 28,
-                         },
-                         "6": Object {
-                           "fontSize": 26,
-                           "lineHeight": 32,
-                         },
-                         "8": Object {
-                           "fontSize": 32,
-                           "lineHeight": 38,
-                         },
-                       },
-                     },
-                   }
-                 }
-                 variant="primaryBlack"
-                 width="100%"
-               >
-                 <View
-                   style={
-                     Array [
-                       Object {
-                         "alignItems": "center",
-                         "display": "flex",
-                         "flexDirection": "row",
-                         "height": "100%",
-                         "justifyContent": "center",
-                         "position": "absolute",
-                         "width": "100%",
-                       },
-                     ]
-                   }
-                 >
-                   <Text
-                     color="#FFF"
-                     fontFamily="Unica77LL-Medium"
-                     fontSize="12px"
-                     lineHeight="16px"
-                     style={
-                       Array [
-                         Object {
-                           "color": "#FFF",
-                           "fontFamily": "Unica77LL-Medium",
-                           "fontSize": 12,
-                           "lineHeight": 16,
-                         },
-                         Object {},
-                       ]
-                     }
-                   >
-                     Pay
-                   </Text>
-                 </View>
-                 <Text
-                   fontFamily="Unica77LL-Medium"
-                   fontSize="12px"
-                   lineHeight="16px"
-                   role="presentation"
-                   style={
-                     Array [
-                       Object {
-                         "fontFamily": "Unica77LL-Medium",
-                         "fontSize": 12,
-                         "lineHeight": 16,
-                       },
-                       Object {
-                         "opacity": 0,
-                       },
-                     ]
-                   }
-                 >
-                   Pay
-                 </Text>
-               </View>
-             </View>
-           </View>
-         </View>
-       </View>
-       <View
          linkStyle={
            Object {
              "color": "#6e1fff",
              "textDecorationLine": "underline",
            }
    at Object.toMatchSnapshot (/home/travis/build/artsy/emission/src/lib/Components/Inbox/Conversations/__tests__/Message-tests.tsx:52:16)
    at Object.asyncJestTest (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/jasmineAsyncInstall.js:102:37)
    at /home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/queueRunner.js:43:12
    at new Promise (<anonymous>)
    at mapper (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/queueRunner.js:26:19)
    at /home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/queueRunner.js:73:41
</code></pre>
</details>
</li>
<br/>

</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: data/complete.queryMap.json [hunk 1]
```diff
@@ -172,5 +172,12 @@
...
+  "58656144a3fed13537098e863b9c7e72": "mutation SendConversationMessageMutation(\n  $input: SendConversationMessageMutationInput!\n) {\n  sendConversationMessage(input: $input) {\n    messageEdge {\n      node {\n        impulse_id: impulseID\n        is_from_user: isFromUser\n        body\n        id\n        ...Message_message\n      }\n    }\n  }\n}\n\nfragment Message_message on Message {\n  body\n  created_at: createdAt\n  is_from_user: isFromUser\n  from {\n    name\n    email\n  }\n  attachments {\n    id\n    internalID\n    content_type: contentType\n    download_url: downloadURL\n    file_name: fileName\n    ...ImagePreview_attachment\n    ...PDFPreview_attachment\n  }\n}\n\nfragment ImagePreview_attachment on Attachment {\n  download_url: downloadURL\n  ...AttachmentPreview_attachment\n}\n\nfragment PDFPreview_attachment on Attachment {\n  file_name: fileName\n  ...AttachmentPreview_attachment\n}\n\nfragment AttachmentPreview_attachment on Attachment {\n  internalID\n}\n",
```
Comment (joeyAghion) [inline data/complete.queryMap.json]: Unrelated changes?

**Comment by ashfurrow** [comment]
@joeyAghion great question – I think that makes sense. The code is checking if `message.invoice` exists, so returning `null` should be safe 👍

**Comment by mzikherman** [comment]
LGTM! RIP to one less special type of attachment preview supported in the app's messaging 🍺

**Comment by artsyit** [comment]
:rocket: PR was released in v1.21.51 :rocket:
